src/lib/snyk-to-html.ts

```diff
@@ -20,6 +24,27 @@ function readFile(filePath: string, encoding: string): Promise<string> {
   });
 }
 
+function handleInvalidJson(reason: any) {
+  if (!reason.isInvalidJson) {
+    reason.message = 'Cannot run `snyk-to-html`. Make sure you are providing the correct parameters.';
+  }
+  console.log(reason.message);
+}
+
+function promisedParseJSON(json) {
+  return new Promise((resolve, reject) => {
+    try {
+      resolve(JSON.parse(json));
```

Comment:
what happens if I have valid json but it is empty?

Reply:
It will generate the html, but with no information.
Should it reject the empty json? Empty json is valid json..